arctic/store/_store.py

```diff
@@ -0,0 +1,12 @@
+class BaseStore(object):
```

Comment:
yes, probably. Read and Write for sure, but I dont know about the others. Not all classes can append (eg Pickle store doesnt seem to have an append).